verif/big_core/tb/big_core_trk.vh

```diff
@@ -0,0 +1,71 @@
+//-----------------------------------------------------------------------------
+// Title            : big_core_trk
+// Project          : riscv_manycore_mesh_fpga
+//-----------------------------------------------------------------------------
+// File             : big_core_trk.vh
+// Original Author  : Daniel Kaufman
+// Code Owner       : 
+// Created          : 01/2023
+//-----------------------------------------------------------------------------
+// Description :
+// this file contain the trakers for big_core_tb   
+//-----------------------------------------------------------------------------
+
```

Comment:
You may use XMR  (cross module reference) in this file to create the instruction 102 & the PC 102